oak-store-document/pom.xml

```diff
@@ -44,8 +44,8 @@
           <instructions>
             <Import-Package>
               <!-- OAK-7182 -->${guava.osgi.import},
-              com.mongodb*;version="[3.8, 4)";resolution:=optional,
-              org.bson*;version="[3.8, 4)";resolution:=optional,
+              com.mongodb*;version="[3.8, 5)";resolution:=optional,
+              org.bson*;version="[3.8, 5)";resolution:=optional,
```

Comment:
No, they aren't compatible. I have upgraded them in this file. The minimum MongoDB Driver version is now 4.7. I thought it could be a lower one like 4.0, but due to some breaking changes introduced in several classes they are not compatible.